reactive/rerunner.go

```diff
@@ -228,7 +228,19 @@ func NewRerunner(ctx context.Context, f ComputeFunc, minRerunInterval time.Durat
 // run performs an actual computation
 func (r *Rerunner) run() {
```

Comment:
Add a comment here `// Wait for the minimum rerun interval. Exit early if the computation is stopped.`